test/spec/core/targeting_spec.js

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
```

Comment:
Regarding the file location... could we make an effort to group the unit tests into one folder? In this case, `tests/spec/core` => `tests/spec/unit/core`

In the current organization, I'd expect to find integration tests (for example, connecting `videoCache` to `prebid-cache`) right next to unit tests like this one. This is bad for automated testing, because we'll want different subsets of [unit tests, integration tests, nightwatch/selenium tests] to run at different times during the dev, PR, and Release processes.